package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@gnosis.pm/safe-apps-react-sdk": "4.2.0",
     "@gnosis.pm/safe-apps-sdk": "6.3.0",
+    "localforage": "^1.10.0",
```

Comment:
this should be put into the tx-builder package.json because it's the only app using this dependency

Reply:
true, totally unintentional :(